### PR TITLE
MongooseIM 3.6.0

### PR DESCRIPTION
**Pull request by michalwski**
This PR is the final touch for the 3.6.0. It updates:
* CHANGELOG.md
* VERSION
* renames migration guide

### CODE REVIEW

**Comment by mongoose-im** [comment]
[7832.7](https://travis-ci.org/esl/MongooseIM/jobs/641359737) / Erlang 22.0 / elasticsearch_and_cassandra_mnesia / 85512c42143a8d6d8e0fa682952fc00f689efe2f
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2606/7832/elasticsearch_and_cassandra_mnesia.22.0)/ [big](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2606/7832/elasticsearch_and_cassandra_mnesia.22.0/big/ct_run.test%40travis-job-87c7c185-e4be-42e3-99f2-cefd9b11f307.2020-01-24_16.48.35/index.html)
**OK: 328** / Failed: 0 / **User-skipped: 28** / Auto-skipped: 0

**Comment by mongoose-im** [comment]
[7840.1](https://travis-ci.org/esl/MongooseIM/jobs/642822847) / Erlang 22.0 / small_tests / 86f9f2818f563e61b4a6afb2f301dd78364a0462
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2606/7840/small_tests.22.0) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2606/7840/small_tests.22.0/small/ct_run.mongooseim@localhost.2020-01-28_12.17.43/lib.mongooseim.logs/run.2020-01-28_12.17.44/suite.log.html)

**Comment by mongoose-im** [comment]
[7851.1](https://travis-ci.org/esl/MongooseIM/jobs/643287993) / Erlang 22.0 / small_tests / 30b0d758070b3d0fed67b088affb10175bb03900
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2606/7851/small_tests.22.0) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2606/7851/small_tests.22.0/small/ct_run.mongooseim@localhost.2020-01-29_10.13.23/lib.mongooseim.logs/run.2020-01-29_10.13.23/suite.log.html)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,98 @@
...
+
+There were some changes to the database schema so please take a look at [Migration guide](doc/migrations/3.5.0_3.6.0.md) when upgrading from previous version.
+
+## All changes
+
+### Added
+
+- System metrics gathering (#2523, #2532, #2550, #2571, #2578, #2580, #2586, #2591, #2594, #2598, #2601, #2603, #2607, #2610, #2612)
+- MAM disco#info to MUC and MUCLight rooms #(2272)
+- Chat markers support to room's REST API (#2274)
```
Comment (NelsonVides) [inline CHANGELOG.md]:
```suggestion
- Chat markers support for the room's REST API (#2274)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,98 @@
...
+- Possibility to close malicious connections without revealing service details (#2304)
+- Ping response time metrics (#2527)
+- Emoji support in VCard's nickname field (#2539)
+- Swagger documentation hosted by MongooseIM (#2543, #2556)
+- Persistent fields from accumulator to offline storage (#2587)
+    - This requires new column in RDBMS, see migration guide
```
[the remark this answers is not in the document]
Comment (gustawlippa) [reply CHANGELOG.md]: Shouldn't it also be `the` migration guide?

**Comment by mongoose-im** [comment]
[7855.1](https://travis-ci.org/esl/MongooseIM/jobs/643376273) / Erlang 22.0 / small_tests / 4baf65418e31c8119e669dd4be9f2186808fca1f
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2606/7855/small_tests.22.0) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2606/7855/small_tests.22.0/small/ct_run.mongooseim@localhost.2020-01-29_15.38.58/lib.mongooseim.logs/run.2020-01-29_15.38.58/suite.log.html)

**Comment by mongoose-im** [comment]
[7864.1](https://travis-ci.org/esl/MongooseIM/jobs/643548396) / Erlang 22.0 / small_tests / 28c704f90155bc4c85b0cf49d260db5b89a0519e
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2606/7864/small_tests.22.0) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2606/7864/small_tests.22.0/small/ct_run.mongooseim@localhost.2020-01-29_21.28.20/lib.mongooseim.logs/run.2020-01-29_21.28.20/suite.log.html)

**Pull request merged**